docs/update-site/eclipse/compositeArtifacts.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?compositeArtifactRepository version='1.0.0'?>
+<repository name='Saros Releases Repository' type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
+    <properties size='0'>
+    </properties>
+    <children size='1'>
+      <child location='https://saros-project.github.io/update-site-artifacts'/>
```

Comment:
We could benefit from having version specific update sites in the future, at which point it would be nice if the basic structure of the update site artifact repository would reflect that (e.g. by splitting it into folders for different versions). But this is nothing we have to worry about right now.